testdata/NEIGH_STATE_TABLE_key_deletion.txt

```diff
@@ -0,0 +1,4 @@
+{
+    "10.0.0.57": {
```

Comment:
>"10.0.0.57" [](http://example.com/codeflow?start=4&length=11)

Since the protocol defined "a list of paths", could you add test case for multiple paths?

Reply:
Yes, I will add for multiple paths.

Reply:
Done